include/z64player.h

```diff
@@ -708,7 +708,7 @@ typedef struct WeaponInfo {
 #define PLAYER_STATE1_2 (1 << 2)
 #define PLAYER_STATE1_3 (1 << 3)
 #define PLAYER_STATE1_HOSTILE_LOCK_ON (1 << 4) // Currently locked onto a hostile actor. Triggers a "battle" variant of many actions.
-#define PLAYER_STATE1_5 (1 << 5)
+#define PLAYER_STATE1_INPUT_DISABLED (1 << 5)
```

Comment:
`// All inputs will be cleared, and therefore ignored, until this flag is unset.`